src/utils/image_plugins/vision.ts

```
@@ -0,0 +1,25 @@
+  return "create" + str.charAt(0).toUpperCase() + str.slice(1);
+};
+
+const processVision = async (params: ImageProcessorParams) => {
+  const { beat, imagePath } = params;
+
+  const handler = new htmlPlugin({});
+
+  await handler[toCreateName(beat.image.name)](beat.image.data, {
+    name: beat.image.name,
+    imageFilePath: imagePath,
+    htmlFilePath: imagePath.replace(/\.png$/, ".html")
```

ESLint

[error] Insert `,`

(prettier/prettier)